morelights/init.lua

```diff
@@ -115,9 +123,31 @@ if morelights.game == "mineclone2" then
     a.stick = "mcl_core:stick"
 end
 
+if morelights.game == "hades_revisited" then
+    a.glass = "hades_core:glass"
+    a.glass_pane = "hades_xpanes:pane_flat"
+    a.steel = "hades_core:steel_ingot"
+    a.copper = "hades_core:copper_ingot"
+    a.tin = "hades_core:tin_ingot"
+    a.crystal_fragment = "hades_core:mese_crystal_fragment"
+    a.dye_dark = "dye:dark_grey"
+    a.dye_light = "dye:white"
+    a.wool_dark = "wool:grey"
+    a.wool_light = "wool:white"
+    a.wood_dark = "hades_trees:jungle_wood"
+    a.stone_block = "hades_core:stone_block"
+    a.sandstone_block = "hades_core:sandstone"
+    a.dirt = "hades_core:dirt"
+    a.grass = "hades_grass:grass_1"
+    a.cotton = "hades_farming:cotton"
+    a.stick = "hades_core:stick"
+end
+
 -- Use basic_materials brass if available, otherwise register our own.
 if minetest.get_modpath("basic_materials") then
     a.brass = "basic_materials:brass_ingot"
+elseif minetest.get_modpath("basic_materials") then
+    a.brass = "hades_extramaterials:brass_ingot"
```

Comment:
Is this a mistake? There doesn't appear to be any mod called `hades_extramaterials`.